services/src/main/resources/locale/addon/Gamification_ar.properties

```diff
@@ -24,7 +24,7 @@ exoplatform.gamification.leaderboard.Pieopen=\u0641\u062A\u062D \u0627\u0644\u06
 exoplatform.gamification.gamificationinformation.Total=\u0645\u062C\u0645\u0648\u0639 \u0627\u0644\u0646\u0642\u0627\u0637
```

Comment:
no need to change this file. Crowdin will do : http://developer.exoplatform.org/#id-translations

Reply:
ok